dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataverse/DataversePage.java

```diff
@@ -154,16 +147,15 @@ public String init() {
     }
 
     public String saveFeaturedDataverse() {
-        UpdateDataverseCommand cmd =
-                new UpdateDataverseCommand(dataverse, null, featuredDataverses.getTarget(), dvRequestService.getDataverseRequest(), null);
 
-        try {
-            dataverse = commandEngine.submit(cmd);
+        Try<Dataverse> saveFeaturedDataverseOperation = Try.of(() -> dataverseSaver.saveFeaturedDataverse(dataverse, featuredDataverses.getTarget()))
+                .onSuccess(saveOperation -> JsfHelper.addFlashSuccessMessage(BundleUtil.getStringFromBundle("dataverse.feature.update")))
```

Comment:
on method success we'll have `dataverse` object as a result. Maybe refactor `saveOperation` to something like `savedDataverse`.